File: testData/src/java8/pkg/TestTryLoop2.java
package pkg;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class TestTryLoop2 {
  private boolean field;

  public void test(File file) {
    while (true) {
      try {
        if (this.field) {
          Scanner scanner = new Scanner(file);

          continue;
        }

        break;
      } catch (FileNotFoundException e) {
        e.printStackTrace();
      }
    }
  }
}

File: plugins/kotlin/testData/src/kt/pkg/TestTryLoop2.kt
package pkg

import java.io.File
import java.io.FileNotFoundException
import java.util.*

class TestTryLoop2 {
    private val field = false

    fun test(file: File) {
        while (true) {
            try {
                if (this.field) {
                    val scanner = Scanner(file)

                    continue
                }

                break
            } catch (e: FileNotFoundException) {
                e.printStackTrace()
            }
        }
    }
}
